client/src/omtLinkProvider.ts

```diff
@@ -0,0 +1,102 @@
+
+import * as path from 'path';
+import { DocumentLink, DocumentLinkProvider, TextDocument } from 'vscode';
+import { Position, Range, Uri, workspace } from 'vscode';
+import { readFileSync } from 'fs';
+
+const MATCHER = /( {4}["']?)(.*\.omt)/;
+const importMatch = /^import:/g;
+const otherDeclareMatch = /^(\w+):/g;
+
+// Geeft een set zogenaamde DocumentLinks terug die VSCode in de editor toont.
+export default class OMTLinkProvider implements DocumentLinkProvider {
+
+    constructor() {
+    }
+
+    public provideDocumentLinks(document: TextDocument): Promise<DocumentLink[]> {
+
+        return Promise.resolve(workspace.findFiles('**/tsconfig**.json').then(
+            (tsConfigs) => {
+                const pathPaths = tsConfigs
+                    // we need to filter for same parent because another config may have some of the same paths
+                    .filter(uri => document.fileName.startsWith(uri.path.substr(0, uri.path.lastIndexOf('/'))))
+                    .map(uri => {
+                        const file = readFileSync(uri.path, 'utf8');
+                        try {
+                            const json = JSON.parse(file);
+                            if (!!(json.compilerOptions) && !!(json.compilerOptions.paths)) {
```

Comment:
```suggestion
                            if (json.compilerOptions && json.compilerOptions.paths) {
```
should work just as well